lib/saml/user_attributes/ssoe.rb

```diff
@@ -197,6 +198,17 @@ def validate!
 
       private
 
+      def should_raise_idme_uuid_error
+        if !idme_uuid
+          if @saml_settings.nil? ||
+             @saml_settings.assertion_consumer_service_url =~ %r{/v1/} && (@authn_context != INBOUND_AUTHN_CONTEXT)
```

Comment:
This is purely a matter of taste, but I like taking complex conditionals and creating private functions with descriptive names in this case, it'd look something like this:
`if @saml_settings.nil? || auth_context_is_v1_and_not_inbound`
Then the function

> def auth_context_is_v1_and_not_inbound
@saml_settings.assertion_consumer_service_url =~ %r{/v1/} && (@authn_context != INBOUND_AUTHN_CONTEXT)
end

Just an idea to think about